config/webpack-config.js

```diff
@@ -118,8 +123,16 @@ const spanishConf = {
   ]
 };
 
+const devConf = {
```

Comment:
So the idea here is that `devConf` is the development configuration on top of all other webpack configs, correct?

Another way to do it would be to set this in https://github.com/cfpb/cfgov-refresh/blob/master/config/environment.js#L24 

```js
// environment.js

…
NODE_ENV = process.env.NODE_ENV
…
```

And then at the top of this file set:

```js
// webpack-config.js

const envvars = require( '../config/environment' ).envvars;
…
const env = envvars.NODE_ENV
…
```

And then at the bottom set:

```js
…
const configExports = {
  commonConf,
  devConf,
  externalConf,
  modernConf,
  onDemandHeaderRawConf,
  appsConf,
  spanishConf
};

if ( env === 'development' ) {
  for (let key in configExports) {
    Object.assign( configExports[key], devConf );
  }
}

module.exports = configExports;
```

Then elsewhere if you want to set an environment flag (`--dev` or `--development` or whatever), that flag would set the value of `process.env.NODE_ENV` and the changes would trickle down where ever they are needed, instead of having a dev flag for each command.

Reply:
ok, I've removed the ability to set the flag when running gulp scripts. You must set the flag using `export NODE_ENV=development` or setting the variable in your `.bash_profile`.

Reply:
Thanks! I'm thinking the direction we'll want to go is to enable overriding `NODE_ENV` in cli-flags.sh (not here and now, but in the future), that way the `./frontend production` or whatever would be the central place to flip environments, and then elsewhere we'd only be referencing the value of `NODE_ENV` for conditional code based on environment. Does that sound like it makes sense?